llvm/lib/Passes/PassBuilder.cpp

```diff
@@ -528,6 +529,106 @@ PassBuilder::PassBuilder(TargetMachine *TM, PipelineTuningOptions PTO,
 #include "llvm/Passes/MachinePassRegistry.def"
     });
   }
+  auto parseLevelParam = [](StringRef P) -> Expected<OptimizationLevel> {
```

Comment:
I just realized we already have `parseOptLevel` in this file, could we use that instead?